public/js/common/react_modal.js

```diff
@@ -13,93 +15,80 @@ export var Modal = React.createClass({
         return {
             courseId: '',
             course: [],
-            sessions: []
+            sessions: [],
+            modalIsOpen: false,
+            courseTitle : '',
+            formatCourse : '',
         };
     },
 
-    render: function () {
-        if (this.state.courseId) {
-            return (
-                <div className='modal fade'>
-                    <div className='modal-dialog'>
-                        <div className='modal-content'>
-                            <div className='modal-header'>
-                                {getCourseTitle(this.state.courseId)}
-                            </div>
-                            <div className='modal-body'>
-                                <Description course={formatCourseName(this.state.courseId)[0]}/>
-                            </div>
-                            <div className='modal-footer'>
-                            </div>
+    openModal: function(newCourse) {
+        if (newCourse == this.state.courseId) {
+                this.setState({modalIsOpen: true});
+        }else {
+            var formatted = formatCourseName(newCourse);
+            var that = this;
+            this.setState({modalIsOpen: true, courseId: newCourse, courseTitle: getCourseTitle(newCourse, formatted)});
 
-                        </div>
-                    </div>
-                </div>
-            );
-        } else {
-            return <div className='modal fade'></div>;
+            $.ajax({
+                    url: 'course',
```

Comment:
@lanaelsanyoura you seem to be using tabs instead of spaces (here, and below). Please double-check all of your changed files and convert any tabs to four spaces.